python/src/cck/examples.py

```diff
@@ -0,0 +1,55 @@
+from pathlib import Path
+
+
+class Examples:
```

Comment:
This file does not need to be included.

Reply:
So for ruby we have a set of helper methods (Identical to these), that allow us to enumerate the CCK proper. So we can just say run all gherkin examples. So if the structure here is intended to be like ruby then this should stay